aws/set_env_vars_file.ps1

```diff
@@ -0,0 +1,44 @@
+# Get vars from flag or env var
```

Comment:
would you mind adding more informative description for this script and how it should be used?

Reply:
good idea, added header